Tests/MongoSwiftSyncTests/SpecTestRunner/TestOperations/CollectionOperations.swift

```diff
@@ -80,18 +81,32 @@ struct Distinct: TestOperation {
     }
 }
 
-struct Find: TestOperation {
-    let session: String?
+struct Find: TestOperation, UnifiedOperationProtocol {
```

Comment:
I personally prefer to duplicate all the operations (even if it involves just copy/pasting) since the two runners are completely separate otherwise, and we'll eventually have to remove the old one anyways as you point out.

Reply:
fine with me, I've reverted the changes to these and introduced duplicates prefixed with `Unified` for now.